File: EvoHtmlToPdf-v8/Samples/WinForms/VB/EvoHtmlToPdfDemo/HTML_to_PDF/Page_Breaks/Avoid_Page_Breaks_Inside_HTML_Elements_Using_CSS_Sample_Code.vb
Imports System.ComponentModel

Imports System.Text

Namespace EvoHtmlToPdfDemo.HTML_to_PDF.Page_Breaks
	Partial Public Class Avoid_Page_Breaks_Inside_HTML_Elements_Using_CSS_Sample_Code
		Inherits UserControl
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Avoid_Page_Breaks_Inside_HTML_Elements_Sample_Code_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			codeSampleWebBrowser.Visible = True
			Dim codeSampleString As String = System.IO.File.ReadAllText("DemoAppFiles\Input\Code_Samples\VbNet\WinForms\HTML_to_PDF\Page_Breaks\Avoid_Page_Breaks_in_CSS.html")
			codeSampleWebBrowser.DocumentText = codeSampleString
		End Sub
	End Class
End Namespace

File: EvoHtmlToPdf-v8/Samples/WinForms/CS/EvoHtmlToPdfDemo/HTML_to_PDF/Page_Breaks/Avoid_Page_Breaks_Inside_HTML_Elements_Using_CSS_Sample_Code.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;

using System.Text;
using System.Windows.Forms;

namespace EvoHtmlToPdfDemo.HTML_to_PDF.Page_Breaks
{
    public partial class Avoid_Page_Breaks_Inside_HTML_Elements_Using_CSS_Sample_Code : UserControl
    {
        public Avoid_Page_Breaks_Inside_HTML_Elements_Using_CSS_Sample_Code()
        {
            InitializeComponent();
        }

        private void Avoid_Page_Breaks_Inside_HTML_Elements_Sample_Code_Load(object sender, EventArgs e)
        {
            codeSampleWebBrowser.Visible = true;
            string codeSampleString = System.IO.File.ReadAllText(@"DemoAppFiles\Input\Code_Samples\CSharp\WinForms\HTML_to_PDF\Page_Breaks\Avoid_Page_Breaks_in_CSS.html");
            codeSampleWebBrowser.DocumentText = codeSampleString;
        }
    }
}
